Makefile.am

```diff
@@ -7,6 +7,7 @@ EXTRA_DIST = pi-blaster.service
 sbin_PROGRAMS = pi-blaster
 pi_blaster_SOURCES = pi-blaster.c mailbox.c
 
+if !DEBIAN_PKG
```

Comment:
You have removed the file `pi-blaster.boot.sh` so it will not work anymore. We need some way to keep it working for people who are not using systemd and not using the debian package.

This [StackOverflow question](http://stackoverflow.com/questions/26837148/autoconf-and-init-scripts) offers two solutions. The first one is very debian specific and you seem to be more familiar with dh-tools than I am. 

The second one suggests how to cp the file in an `install-exec-hook`. We could use that.